<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Init</class>
 <widget class="QWidget" name="Tab_Init">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Composition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1" colspan="4">
       <widget class="QListWidget" name="list_composition"/>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QLineEdit" name="edit_composition"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_14">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Cell composition:</string>
        </property>
        <property name="buddy">
         <cstring>edit_composition</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Structural Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2" rowspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="cb_shortestInteratomicDistance">
        <property name="toolTip">
         <string>Use this to limit the shortest interatomic distance between atoms. This ensures that atoms aren't overlapping.</string>
        </property>
        <property name="text">
         <string>Limit interatomic distance? (Ã…)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="spin_shortestInteratomicDistance">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Use this to limit the shortest interatomic distance between atoms. This ensures that atoms aren't overlapping.</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_composition</tabstop>
  <tabstop>list_composition</tabstop>
  <tabstop>cb_shortestInteratomicDistance</tabstop>
  <tabstop>spin_shortestInteratomicDistance</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cb_shortestInteratomicDistance</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_shortestInteratomicDistance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
